contracts/polygon/connectors/hop/helpers.sol

```diff
@@ -30,11 +31,27 @@ contract Helpers is DSMath, Basic {
 		uint256 sourceDeadline;
 		uint256 destinationAmountOutMin;
 		uint256 destinationDeadline;
+		bool isWrapped;
```

Comment:
@pradyuman-verma Why send extra param? Instead, we can check this on smart contract only, right?